bl31/bl31.mk

```diff
@@ -12,6 +12,7 @@ BL31_SOURCES		+=	bl31/bl31_main.c				\
 				bl31/aarch64/runtime_exceptions.S		\
 				bl31/aarch64/crash_reporting.S			\
 				bl31/bl31_context_mgmt.c			\
+				bl31/plat_params.c				\
```

Comment:
Do you mean that your mechanism likely won't be parsed before the MMU is enabled? But that would mean that it also cannot be used to pass serial console configuration information in general, right? Is that intended? Or do you expect to enable the MMU before the serial console in the future?

If what you're building is just fundamentally not suited for the coreboot table pointer, we should maybe just special-case it and find a different solution (it's just one pointer, after all). How about adding a new, optional field to bl31_params_t, for example? I would really like to find a solution where not every platform has to come up with its own custom way of fitting this into the bl31_plat_params somehow.